flink-formats/flink-avro/src/test/java/org/apache/flink/table/runtime/batch/AvroTypesITCase.java

```diff
@@ -147,24 +150,26 @@ public AvroTypesITCase(
 
 	@Test
 	public void testAvroToRow() throws Exception {
-		ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 		env.getConfig().registerTypeWithKryoSerializer(LocalDate.class, AvroKryoSerializerUtils.JodaLocalDateSerializer.class);
 		env.getConfig().registerTypeWithKryoSerializer(LocalTime.class, AvroKryoSerializerUtils.JodaLocalTimeSerializer.class);
-		BatchTableEnvironment tEnv = BatchTableEnvironment.create(env, config());
+		StreamTableEnvironment tEnv = StreamTableEnvironment.create(env, EnvironmentSettings.newInstance().useBlinkPlanner().build());
 
-		Table t = tEnv.fromDataSet(testData(env));
+		Table t = tEnv.fromDataStream(testData(env));
 		Table result = t.select($("*"));
 
-		List<Row> results = tEnv.toDataSet(result, Row.class).collect();
+		Iterable<Row> users = () -> DataStreamUtils.collect(tEnv.toAppendStream(result, Row.class));
+		List<Row> results = StreamSupport
```

Comment:
nit: We recently added a `CollectionUtils.iterableToList` maybe this is easier to read than the StreamSupport class.